dts/st/h7/stm32h743zitx-pinctrl.dtsi

```diff
@@ -3354,4 +3371,4 @@
 
 		};
 	};
-};
\ No newline at end of file
+};
```

Comment:
Your editor is not properly configured and adds new line at the end of the files when saving.